libs/stache/src/lib/modules/wrapper/wrapper.component.scss

```diff
@@ -1,70 +1,31 @@
 @import '../../styles/_variables';
 
-::ng-deep .stache-wrapper {
+stache-wrapper {
   background-color: #fff;
-  line-height: $stache-line-height-base;
 
-  p:not([class]),
```

Comment:
👇 All of these styles are provided by our SKY UX Theme stylesheet, so we should remove these overrides.